File: rust/7_kyu/area_of_an_arrow.rs
/*
SPDX-FileCopyrightText: 2025 Michael Persico <michael.a.persico@gmail.com>
 
SPDX-License-Identifier: MIT-0
*/

// https://www.codewars.com/kata/589478160c0f8a40870000bc

fn arrow_area(a: u32, b: u32) -> f64 {
    (a as f64 * (b as f64)/2.0)/2.0
}

fn main() {
    println!("{}", arrow_area(7,6)); // 10.5
}

File: go/7_kyu/area_of_an_arrow.go
/*
SPDX-FileCopyrightText: 2025 Michael Persico <michael.a.persico@gmail.com>

SPDX-License-Identifier: MIT-0
*/

// https://www.codewars.com/kata/589478160c0f8a40870000bc

package main

import "fmt"


func ArrowArea(a, b int) float64 {
	return float64(a*b) / 4
}

func main() {
	fmt.Println(ArrowArea(4, 2)) // 2
}